evalml/data_checks/highly_null_data_check.py

```diff
@@ -12,17 +12,25 @@ class HighlyNullDataCheck(DataCheck):
     """Checks if there are any highly-null columns and rows in the input.
 
     Arguments:
-        pct_null_threshold(float): If the percentage of NaN values in an input feature exceeds this amount,
-            that column/row will be considered highly-null. Defaults to 0.95.
+        pct_null_col_threshold(float): If the percentage of NaN values in an input feature exceeds this amount,
+            that column will be considered highly-null. Defaults to 0.95.
+        pct_null_row_threshold(float): If the percentage of NaN values in an input feature exceeds this amount,
+            that row will be considered highly-null. Defaults to 0.95.
 
     """
 
-    def __init__(self, pct_null_threshold=0.95):
-        if pct_null_threshold < 0 or pct_null_threshold > 1:
+    def __init__(self, pct_null_col_threshold=0.95, pct_null_row_threshold=0.95):
+        if (
+            pct_null_col_threshold < 0
+            or pct_null_col_threshold > 1
+            or pct_null_row_threshold < 0
+            or pct_null_row_threshold > 1
+        ):
             raise ValueError(
-                "pct_null_threshold must be a float between 0 and 1, inclusive."
+                "pct null thresholds must be a float between 0 and 1, inclusive."
```

Comment:
I think it'd be nice to separate the checks for cols vs rows to raise error specifically on the value needed